scripts/pylintrc_default

```diff
@@ -91,10 +110,59 @@ load-plugins=pylint.extensions.check_docs
 # - no-name-in-module: Error gives a lot of false positives for names which
 #                      are defined dynamically. Also, any truly missing names
 #                      will be detected by our 100% code coverage.
-#
-# New opinions in pylint 1.6, enforcing PEP 257.  #1968 for eventual fixes
-# - missing-raises-doc
+# - missing-raises-doc: New in PyLint 1.6, enforcing PEP 257 and flagged our
+#                       existing codebase.  See #1968 for eventual fixes.
 disable =
+    import-star-module-level,
+    old-octal-literal,
+    oct-method,
+    print-statement,
+    unpacking-in-except,
+    parameter-unpacking,
+    backtick,
+    old-raise-syntax,
+    old-ne-operator,
+    long-suffix,
+    dict-view-method,
+    dict-iter-method,
+    metaclass-assignment,
+    next-method-called,
+    raising-string,
+    indexing-exception,
+    raw_input-builtin,
+    long-builtin,
+    file-builtin,
+    execfile-builtin,
+    coerce-builtin,
+    cmp-builtin,
+    buffer-builtin,
+    basestring-builtin,
+    apply-builtin,
+    filter-builtin-not-iterating,
+    using-cmp-argument,
+    useless-suppression,
+    range-builtin-not-iterating,
+    suppressed-message,
+    no-absolute-import,
+    old-division,
+    cmp-method,
+    reload-builtin,
+    zip-builtin-not-iterating,
+    intern-builtin,
+    unichr-builtin,
+    reduce-builtin,
+    standarderror-builtin,
+    unicode-builtin,
+    xrange-builtin,
+    coerce-method,
+    delslice-method,
+    getslice-method,
+    setslice-method,
+    input-builtin,
+    round-builtin,
+    hex-method,
+    nonzero-method,
+    map-builtin-not-iterating,
```

Comment:
They are the default. See the commits on this PR.

In particular: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2396/commits/a03e1c5f0c000700457b0542c40815d1575c3158